src/components/molecules/TextView/TextViewRecordLargeNumbers.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import Box from '@material-ui/core/Box';
+import { roadIconColors, silverSmokeColor } from 'style';
+import SeverityImage from './SeverityImage';
+import { makeStyles } from '@material-ui/core/styles';
+import { Typography } from 'components/atoms';
+
+interface IProps {
+  numOfAccidents: number;
+  severityDesc: string;
+  imgSrc: string;
+  isLast?: boolean;
+}
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    display: 'flex',
+    justifyContent: 'space-around',
+    width: '80%',
+    height: '20%',
+  },
+  text: {
+    top: theme.spacing(2),
+    display: 'flex',
+    alignItems: 'center',
+    textAlign: 'center',
+    justifyContent: 'center',
+    width: '30%',
+  },
+  acNum: {
+    position: 'relative',
+    display: 'flex',
+    alignItems: 'center',
+    textAlign: 'center',
+    justifyContent: 'center',
+    color: roadIconColors.red,
+    fontWeight: 'bold',
+    fontSize: '350%',
```

Comment:
use `Typography` component  instead of `fontSize` property . if the size is not existed in `Typography` variants (`Title1`, `Body1` etc..), create a new one in `theme.ts`